src/components/Activities/Activity1/Game2/components/GameSlide.tsx

```diff
@@ -30,15 +32,26 @@ function GameSlide(props: GameSlideProps): JSX.Element {
     return props.textDefault ? props.textDefault : '';
   };
 
+  const handleAlienState = (state: ALIEN_STATE) => {
+    clearTimeout(alienTimeout.current);
```

Comment:
and this helps it chill out from the Timeout typing change on my side!
```suggestion
    alienTimeout.current && clearTimeout(alienTimeout.current);
```

Reply:
Iirc if it gets undefined, nothing will happen?

Reply:
i think so? wdym? confuse (it was jsut that if i did `clearTimeout(*****undefined****)`, my vscode was angry
![image](https://user-images.githubusercontent.com/29292203/112382139-b68cb080-8ca8-11eb-8efc-915dcf55507c.png)

Reply:
actually maybe my computer is just mad